app/services/authenticate_user.rb

```diff
@@ -0,0 +1,28 @@
+class AuthenticateUser < Patterns::Service
+  attr_accessor :errors
+
+  def initialize(email, password)
+    @email = email
+    @password = password
+    @errors = {}
+  end
+  
+  def call
+    JsonWebToken.encode(user_id: user.id) if user
+  end
+ 
+  def success?
```

Comment:
errors and success? functions should actually be moved into either the patterns gem, or to some base command object, since it's useful for multiple objects.